ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Agate styled button components and behaviors.
+ *
+ * @example
+ * <Button small>Hello Enact!</Button>
+ *
+ * @module agate/Button
+ * @exports Button
+ * @exports ButtonBase
+ * @exports ButtonDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Pure from '@enact/ui/internal/Pure';
+import Toggleable from '@enact/ui/Toggleable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Button from '../Button';
+// import {MarqueeDecorator} from '../Marquee';
+import Skinnable from '../Skinnable';
+
+import css from './ToggleButton.less';
+
+
+/**
+ * A stateless [Button]{@link agate/Button.Button} that can be toggled by changing its
+ * `selected` property.
+ *
+ * @class ToggleButtonBase
+ * @memberof agate/ToggleButton
+ * @extends agate/Button.Button
+ * @ui
+ * @public
+ */
+const ToggleButtonBase = kind({
+	name: 'ToggleButton',
+
+	propTypes: /** @lends agate/Button.ButtonBase.prototype */ {
+		/**
+		 * The background-color opacity of this button.
+		 *
+		 * * Values: `'translucent'`, `'lightTranslucent'`, `'transparent'`
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		backgroundOpacity: PropTypes.oneOf(['translucent', 'lightTranslucent', 'transparent']),
```

Comment:
Do we have styles defined for these values? I don't see them or notice visual changes when I change these values. It appears these would be passed to `Button` but it's not clear to me that they're handled there either.

Reply:
woops we don't need this for ToggleButton